.nuspec/Microsoft.Maui.Controls.DefaultItems.props

```diff
@@ -18,5 +18,5 @@
     <MauiCss Include="**\*.css" Exclude="$(DefaultItemExcludes);$(DefaultExcludesInProjectFolder);$(DefaultWebContentItemExcludes)" />
   </ItemGroup>
 
-  <Import Project="Microsoft.Maui.Controls.SingleProject.props" />
+  <Import Project="Microsoft.Maui.Controls.SingleProject.AutoImport.props" />
```

Comment:
Renamed the file in the repo and reused the old name for a file that we want to ship.